ui/values/dimensions.go

```diff
@@ -28,6 +28,9 @@ var (
 	TextSize28 = unit.Dp(28)
 
 	CopyBtnAlignment          = unit.Dp(-5)
+	TabTopPadding             = unit.Dp(-10)
 	WalletSyncBoxContentWidth = unit.Dp(280)
 	AccountLineWidth          = unit.Dp(350)
+	OutLinePadding            = unit.Dp(1)
+	BorderWidth               = unit.Px(2)
```

Comment:
The paddings and margins should follow the normal padding convention. Example: "MarginPadding5". For the negative values, you can use underscore. So, "MarginPadding_5".